pyccel/ast/operators.py

```diff
@@ -488,6 +498,14 @@ class PyccelMinus(PyccelArithmeticOperator):
     """
     _precedence = 12
 
+    def __new__(cls, arg1, arg2):
+        if isinstance(arg1, LiteralFloat) and \
+           isinstance(arg2, LiteralComplex) and \
+           arg2.real_part == LiteralFloat(0):
+            return LiteralComplex(arg1, -float(arg2.imag_part))
```

Comment:
Above you used the properties `real` and `imag`, here you are using `real_part` and `imag_part` instead. What's the difference?

Reply:
`real_part` and `imag_part` are the properties of `PythonComplex` while `real` and `imag` are the properties of `LiteralComplex`. We should probably use the same for both. Which do you prefer?

Reply:
I prefer the shortest names: `real` and `imag`